draft-ietf-rats-msg-wrap.md

```diff
@@ -308,6 +309,7 @@ A single Collection CMW can contain a mix of different message types, and it can
 The Collection CMW ({{fig-cddl-collection}}) is defined as a CBOR map or JSON object containing CMW values.
 The position of a `cmw` entry in the `cmw-collection` is not significant.
 Labels can be strings (or integers in the CBOR serialization) that serve as a mnemonic for different conceptual messages in the Collection.
+The character set allowed in the string representation of a label is the "Unicode Scalars" defined in {{Section 4.1 of -unicode-subsets}}.
```

Comment:
I apologize if this input makes no sense; I don't understand the RATS space.  Is a `text-label` expected to be seen by humans? If so, the choice of Unicode Scalars means that a value such as. `"\u0000\u007F\uAFFFE"` is allowed (the characters are NUL, DEL, and a noncharacter).  If that's OK, fine, but if not, consider using the "Unicode Assignables" subset from 9839.

Reply:
While human readability is an advantage, the main requirement is that they are valid JSON object or CBOR map keys.
If you think “Unicode assignables” would be a better fit, we can make the change.

Reply:
It really depends if these things might need to be displayed to a human.  If not, then Scalars is a good choice, exclude the surrogates (they can break some software) and you're done.  If it would be advantageous to show them to people sometimes, Assignables really would be a better choice, because `"\u0000\u007F\uAFFFE"` is just not going to be legible to human eyes.

Reply:
Well, this contaminates the present document with Unicode esoterics.  It is the job of people defining a name to make that useful, "assignables" is not going to help with that at all.